### PR TITLE
Temporary images refactor

### PR DESCRIPTION
**Pull request by Pagendera**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -37,7 +45,15 @@ new Swiper('.dishes-slider', {
...
\ No newline at end of file
+});
+
+document.querySelector('#advantages-image').src=advantages
+document.querySelector('#questions-image').src=questions
+document.querySelector('#pizza').src=pizza
+document.querySelector('#burger').src=burger
+document.querySelector('#sushi').src=sushi
+document.querySelector('#cookies').src=cookies
+document.querySelector('#puncake').src=puncake
```
Comment (neodavis) [inline src/main.js:59]: leave spaces between property name and "=" operator

File: src/style.css [hunk 2]
```diff
@@ -47,5 +48,12 @@ body {
...
-    height: 500px;
+    height: 600px;
 }
+
+.swiper-pagination-bullet{
+    box-shadow: 0 0 30px black;
+}
+
+
+
```
Comment (neodavis) [inline src/style.css:59]: get rid off unnecessary breaks at the end of the file (always leave only 1 empty line)

File: src/main.js [hunk 3]
```diff
@@ -3,17 +3,25 @@ import 'wow.js/css/libs/animate.css';
 import '@fortawesome/fontawesome-free/css/all.min.css';
 import 'swiper/css/bundle';
 
+import advantages from '../assets/advantages-image.png'
+import questions from '../assets/questions-image.png'
+import pizza from '../assets/pizza.png'
+import burger from '../assets/burger.png'
+import sushi from '../assets/sushi.png'
+import cookies from '../assets/cookies.png'
+import puncake from '../assets/puncake.png'
```
Comment (neodavis) [inline src/main.js:12]: please, always provide semi-columns on the end of the line (keep code clean, check other places for possible improvement)

**Review by neodavis — CHANGES_REQUESTED** [review]
Please, resolve comments bellow + replace pictures with compressed versions (images load is so slow for now)

**Review by ashotvdovychenko — APPROVED** [review]
I have no questions

**Review by neodavis — APPROVED** [review]

**Pull request merged**